kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/topics/TopicTests.java

```diff
@@ -1,34 +1,96 @@
+    public static final String UPDATED_TIME_TO_RETAIN_VALUE = "604800001";
+    public static final String UPDATED_MAX_SIZE_ON_DISK = "20 GB";
+    public static final String UPDATED_MAX_MESSAGE_BYTES = "1000020";
+
+    @BeforeAll
+    @SneakyThrows
+    public static void beforeAll() {
+        Helpers.INSTANCE.apiHelper.createTopic(SECOND_LOCAL, UPDATE_TOPIC);
+        Helpers.INSTANCE.apiHelper.createTopic(SECOND_LOCAL, DELETE_TOPIC);
+    }
 
-    @AfterEach
+    @AfterAll
     @SneakyThrows
-        void  afterEach(){
-            helpers.apiHelper.deleteTopic("secondLocal","new-topic");
+    public static void afterAll() {
+        Helpers.INSTANCE.apiHelper.deleteTopic(SECOND_LOCAL, UPDATE_TOPIC);
+        Helpers.INSTANCE.apiHelper.deleteTopic(SECOND_LOCAL, DELETE_TOPIC);
     }
 
     @SneakyThrows
     @DisplayName("should create a topic")
     @Test
-    void createTopic(){
-        helpers.apiHelper.createTopic("secondLocal","new-topic");
-        pages.open()
-                .mainPage.shouldBeOnPage()
-        .goToSideMenu("secondLocal", MainPage.SideMenuOptions.TOPICS)
-        .shouldBeTopic(NEW_TOPIC);
+    void createTopic() {
+        try {
+            helpers.apiHelper.createTopic(SECOND_LOCAL, NEW_TOPIC);
+            pages.open()
+                    .shouldBeOnPage()
+                    .goToSideMenu(SECOND_LOCAL, MainPage.SideMenuOptions.TOPICS)
+                    .shouldBeTopic(NEW_TOPIC);
+        } finally {
+            helpers.apiHelper.deleteTopic(SECOND_LOCAL, NEW_TOPIC);
+        }
     }
+
+    @SneakyThrows
+    @DisplayName("should update a topic")
+    @Test
+    void updateTopic(){
+        final String path = "ui/clusters/" + SECOND_LOCAL + "/topics/" + UPDATE_TOPIC;
+
+        pages.openTopicsListPage()
+                .shouldBeOnPage()
+                .openTopic(UPDATE_TOPIC);
+        pages.openTopicViewPage(path)
+                .openEditSettings()
+                .changeCleanupPolicy(COMPACT_POLICY_VALUE)
+                .changeTimeToRetainValue(UPDATED_TIME_TO_RETAIN_VALUE)
+                .changeMaxSizeOnDisk(UPDATED_MAX_SIZE_ON_DISK)
+                .changeMaxMessageBytes(UPDATED_MAX_MESSAGE_BYTES)
+                .submitSettingChanges();
+        pages.reloadPage();
+        TopicViewPage topicViewPage = pages.openTopicViewPage(path)
+                .openEditSettings();
+
+        waitForSelectedValue(topicViewPage.cleanupPolicy, COMPACT_POLICY_VALUE);
+
+        Assertions.assertEquals(UPDATED_TIME_TO_RETAIN_VALUE,topicViewPage.timeToRetain.getValue());
+        Assertions.assertEquals(UPDATED_MAX_SIZE_ON_DISK,topicViewPage.maxSizeOnDisk.getSelectedText());
+        Assertions.assertEquals(UPDATED_MAX_MESSAGE_BYTES,topicViewPage.maxMessageBytes.getValue());
+    }
+
+    @SneakyThrows
+    @DisplayName("should delete topic")
+    @Test
+    @Disabled
+    void deleteTopic(){
+        final String path = "ui/clusters/" + SECOND_LOCAL + "/topics/" + DELETE_TOPIC;
```

Comment:
small suggestion here. `DELETE_TOPIC` sounds like action,maybe `TOPIC_TO_DELETE` or something like that?

Reply:
Done!